client/app/main/main.js

```diff
@@ -5,5 +5,15 @@ angular.module('maerkApp')
     $stateProvider.state('main', {
       url: '/',
       template: '<main class="flex layout column"></main>'
```

Comment:
Change this route to be abstact.
 If main route is not abstract you will have an error when reloading